README.md

```diff
@@ -118,3 +118,11 @@ val t : Topojson.t = <abstr>
 - : string =
 "{\"type\":\"Topology\",\"objects\":{\"example\":{\"type\":\"LineString\",\"arcs\":[0],\"foreign\":\"8\"}},\"arcs\":[[[0,0]]]}"
 ```
+
+#### Example usage of foreign_members
+
+Given the Topojson Object above, we can extract the foreign members by using the foreign_member function in the Topojson.Geometry module.
+
+```ocaml
+let foreign_member_usage = Topojson.Geometry.foreign_members topojson
```

Comment:
This is great thanks, but I think it would be better to use MDX's top-level syntax so it prints the value too:

```suggestion
# Topojson.Geometry.foreign_members topojson
```

Reply:
Okay. Done